<html>
<head>
<title>BLink</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
 <link href="/css/normal.css" rel="stylesheet" type="text/css">
</head>

<body bgcolor="#FFFFFF" text="#000000">
<p><font size="4"><b><font size="5">Users' documentation for BLink in c++</font></b></font></p>
<p><br>
  <b><font size="4">Installation</font></b></p>
<p><b>Compiled version:</b></p>
<p>Compiled version for linux.-After you have succesfully obtained the archive 
  you can execute the following Unix commands to install the program:</p>
<ul>
  <li>gunzip BLink2.0.gz</li>
</ul>
<p><b>Source code version:</b></p>
<p>Source code version for linux.-After you have succesfully obtained the archive 
  you can execute the following Unix commands to install the program:</p>
<ul>
  <li>tar -xvzf BLink2.0.tar.gz</li>
  <li>cd BLink2.0</li>
  <li>make BLink</li>
</ul>
<p><font size="4"><b>Execution</b></font></p>
<p>To use BLink, use the following command line:</p>
<p>./BLink &lt;filename&gt; [&lt;isPhased (0: no default, 1: yes)&gt;] [&lt;useBayesianMeasure (1: yes default, 0:no)&gt;]</p>
<p>where &lt;filename&gt; is the input file with format as shown below.</p>
<p>By default, input data are considered unphased and the Bayesian method is used. Use 1 as a second argument is the phase is solved (alleles are ordered).
Use 0 as a third argument if the original (MLE) LD measures are to be computed.
<p>&nbsp;</p>

<p><b>Format of the input genotype file</b> </p>
<p>A text file of SNP data set, in <b>post-makeped format</b> as described here:</p>
<p>One row per individual, 7 + 2x<i>TotalSNPs</i>
  columns, with <i>TotalSNPs</i> being the number of SNPs. Columns must be separared
  by a blank or tab character.Each column must contain an integer.
  <br>  <a href=example.gou>See an example of input file</a>.
  </p>
<p>Description of columns: </p>
<ul>
  <li>Column 1: family code </li>
  <li>Column 2: individual code </li>
  <li>Column 3: father code (0 if the individual is not a child)</li>
  <li>Column 4: mother code (0 if the individual is not a child)</li>
  <li>Column 5: gender (not used by rTDT)</li>
  <li>Column 6: affection status (not used by rTDT)</li>
  <li>Column 7: another individual id (not used by rTDT) </li>
  <li>Columns 8 until 7+2*<i>TotalSNPs</i>: SNPs values (each SNP uses two contiguous
    columns). Example: if SNP1 value for an individual is 12 (heterozygous), column8
    will have the value 1 and column9 will have the 2.</li>
</ul>

<p><b>Format of the input file with physical positions</b>
<p>One position per row. <a href="example.pou">Click here to see an example</a>.</p>
<p><b>Output for the program BLink</b></p>
<p>The output will be written at a file with the same name as the input file 
  but with the extension &quot;.pm&quot;. The output file is a csv text file with one line for each pair of SNPs with the number and positions of each snp, haplotype and allele frequencies and the Bayesian estimation of D, D', r2 and Yules Q.</p>
</body>
</html>
